gtest/src/wasm_executor.rs

```diff
@@ -0,0 +1,349 @@
+use crate::empty_ext::ExtImplementedStruct;
+use core_processor::common::JournalNote;
+use gear_backend_wasmtime::{env::StoreData, funcs::FuncsHandler};
+use gear_core::{
+    env::{Ext, ExtCarrier},
+    ids::ProgramId,
+    memory::{PageBuf, PageNumber, WasmPageNumber},
+    message::IncomingMessage,
+    program::Program,
+};
+use std::{collections::BTreeMap, convert::TryInto};
+use wasmtime::{
+    Config, Engine, Extern, Func, Instance, Memory as WasmtimeMemory, MemoryType, Module, Store,
+    Val,
+};
+
+#[allow(dead_code)]
```

Comment:
Why it can be dead if it is a part of public API?

Reply:
I add them to disable warnings about unused code. I will delete it when pr will be on the final stage